Error: No named parameter with the name 'body1'.

lib/main.dart:17:45: Error: No named parameter with the name 'body1'.
      theme: ThemeData(textTheme: TextTheme(body1: TextStyle(fontFamily: 'Poppins'),),),
                                            ^^^^^
/C:/src/flutter/flutter/packages/flutter/lib/src/material/text_theme.dart:81:9: Context: Found this candidate, but the arguments don't match.
  const TextTheme({
        ^^^^^^^^^


FAILURE: Build failed with an exception.
* Where:
Script 'C:\src\flutter\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 1070

--- a/lib/main.dart
+++ b/lib/main.dart
@@ -13,18 +13,23 @@ class ChatterApp extends StatelessWidget {
     return MaterialApp(
       debugShowCheckedModeBanner: false,
       title: 'Chatter',
-      
-      theme: ThemeData(textTheme: TextTheme(body1: TextStyle(fontFamily: 'Poppins'),),),
+
+      theme: ThemeData(
+        textTheme: TextTheme(
+          bodyText1: TextStyle(
+            fontFamily: 'Poppins',
+          ),
+        ),
+      ),
       // home: ChatterHome(),
       initialRoute: '/',
       routes: {
-        '/':(context)=>ChatterHome(),
-        '/login':(context)=>ChatterLogin(),
-        '/signup':(context)=>ChatterSignUp(),
-        '/chat':(context)=>ChatterScreen(),
+        '/': (context) => ChatterHome(),
+        '/login': (context) => ChatterLogin(),
+        '/signup': (context) => ChatterSignUp(),
+        '/chat': (context) => ChatterScreen(),
         // '/chats':(context)=>ChatterScreen()
       },
     );
   }
 }
-

--- a/lib/pages/chat.dart
+++ b/lib/pages/chat.dart
@@ -63,7 +63,7 @@ class _ChatScreenState extends State<ChatScreen> {
                       decoration: kMessageTextFieldDecoration,
                     ),
                   ),
-                  FlatButton(
+                  TextButton(
                     onPressed: () {
                       //Implement send functionality.
                     },

